packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -148,6 +156,141 @@ export class LitNodeClientNodeJs extends LitCore {
+    const _expiration =
+      expiration ?? new Date(Date.now() + 1000 * 60 * 7).toISOString();
+    const statement = '';
+
+    // -- if it's not ready yet, then connect
+    if (!this.ready) {
+      await this.connect();
+    }
+
+    // -- validate if rliTokenId is empty
+    if (!rliTokenId) {
+      throw new Error('rliTokenId must exist');
+    }
+
+    // -- validate
+    if (!dAppOwnerWallet || !addresses) {
+      throw new Error('Both parameters must exist');
+    }
+
+    // -- validate dAppOwnerWallet is an ethers wallet
+    // if (!(dAppOwnerWallet instanceof ethers.Wallet || ethers.Signer)) {
+    //   throw new Error('dAppOwnerWallet must be an ethers wallet');
+    // }
+
+    // -- validate addresses has to be an array and has to have at least one address
+    if (!Array.isArray(addresses) || addresses.length === 0) {
+      throw new Error(
+        'Addresses must be an array and has to have at least one'
+      );
+    }
+
+    // -- validate use is set
+    if (!uses) {
+      throw new Error('uses must be set');
+    }
+
+    // -- create LitRLIResource (note: we have other resources such as LitAccessControlConditionResource, LitPKPResource and LitActionResource)
+    // lit-ratelimitincrease://{tokenId}
+    const litResource = new LitRLIResource(rliTokenId);
+    // console.log('litResource:', litResource);
+
+    // Strip the 0x prefix from each element in the addresses array
+    addresses = addresses.map((address) =>
+      address.startsWith('0x') ? address.slice(2) : address
+    );
+
+    const recapObject = await this.generateSessionCapabilityObjectWithWildcards(
+      [litResource]
+    );
+
+    recapObject.addCapabilityForResource(
+      litResource,
+      LitAbility.RateLimitIncreaseAuth,
+      { nft_id: [rliTokenId], delegate_to: addresses, uses: uses.toString() }
+    );
+
+    // console.log('recapObject:', recapObject);
+    // console.log('attenuations:', JSON.stringify(recapObject.attenuations));
+
+    // make sure that the resource is added to the recapObject
+    const verified = recapObject.verifyCapabilitiesForResource(
+      litResource,
+      LitAbility.RateLimitIncreaseAuth
+    );
+
+    // -- validate
+    if (!verified) {
+      throw new Error('Failed to verify capabilities for resource');
+    }
+
+    // console.log('verified:', verified);
+
+    let nonce = this.getLatestBlockhash();
+
+    // -- get auth sig
+    let siweMessage = new siwe.SiweMessage({
+      domain: _domain,
+      address: dAppOwnerWalletAddress,
+      statement,
+      uri: ORIGIN,
+      version: '1',
+      chainId: 1,
+      nonce: nonce?.toString(),
+      expirationTime: _expiration,
+    });
+
+    siweMessage = recapObject.addToSiweMessage(siweMessage);
+    // console.log('XX siweMessage:', siweMessage);
+
+    let messageToSign = siweMessage.prepareMessage();
+    // console.log('messageToSign:', messageToSign);
+
+    const signature = await dAppOwnerWallet.signMessage(messageToSign);
```

Comment:
todo: refactor to support other signing methods (providers through `request` or `call` as an example)

Reply:
https://linear.app/litprotocol/issue/LIT-2321/[js-sdk]-add-browser-support-for-createrlidelegationauthsig